pages/talents/[id]/ProfilePage.tsx

```diff
@@ -139,8 +134,10 @@ const GET_ALL_INFO = gql`
   }
 `;
 
-const ProfilePage = ({ t }: PageProps): React.ReactElement => {
-  const { data: talentIds, loading: idLoading } = useQuery(GET_ALL_TALENTS);
+const ProfilePage = ({
+  t,
+  editable,
```

Comment:
I am not sure if I understand the `editable` correctly. I understand that we want to have a "view-only" and an "editable" mode. 

But how do we plan to achieve this in the component? If the component is being rendered with `editable=false`, how do we plan to make sure that the user cannot edit anything?

Initially, I thought that we are going to take care of this in our resolvers, i.e. check the user and his permissions in our resolvers. And only run the query if he is allowed to do that.

Reply:
Both. In the query, but in the frontend we also only want to show the 'edit' button, if the user can actually edit. That's what editable is for: we only show the edit buttons if it is set to true.

Reply:
Ah okay, understood.
Makes sense to me.
But then we also need to add authorization to the queries.
Do we want to do this in a separate ticket?